packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -254,6 +272,10 @@ contract PermissionManager is Initializable {
         bytes32 _permissionId,
         IPermissionOracle _oracle
     ) internal {
+        if (_permissionId == ROOT_PERMISSION_ID && _who == ANY_ADDR) {
+            revert RootPermissionForAnyAddressDisallowed();
+        }
```

Comment:
Precisely in the case of an Oracle, I think it does make sense to be able to grant permissions to anyone. As explained on Discord, this would be as if the Linux Kernel disallowed the `root` user from setting `777` permissions to `/etc/password` if desired. 

Happy to hear other opinions, but IMO the UI/SDK should be the ones alerting DAO members of the potential consequences, other than restricting a feature from the protocol level. 

```suggestion
```

Reply:
I strongly agree

Reply:
In case this get's reverted, remember to remove the `error RootPermissionForAnyAddressDisallowed` that was created as well.